2023-04-17-report.org

```diff
@@ -0,0 +1,97 @@
+
+*** Milestone Progress
+
+Philip and Arthur took some time this week to sort through this year's GSoC proposals and review them as well as rank them. We have landed on our final selection, and are now waiting to hear from Google. We look forward to working with you all over the summer and would like to thank you for the effort put into these proposals, as well as the very high number we received!
+
+We have added new milestones to the project's repository in order to better split the upcoming months of work on the compiler, and are listing them below. We will keep pushing to get libcore compiling as soon as possible and look forward to the upcoming challenges!
+
+** Completed Activities
+
+- Unify HIR::ExprStmt deriving classes [[https://github.com/rust-gcc/gccrs/pull/2111][PR2111]]
+- Handle underscore in MacroMatch. [[https://github.com/rust-gcc/gccrs/pull/2109][PR2109]]
+- Fix StructExprStructFields tokenstream [[https://github.com/rust-gcc/gccrs/pull/2100][PR2100]]
+- parser: macro: reject separator in `?` repetition [[https://github.com/rust-gcc/gccrs/pull/2098][PR2098]]
+- Unify AST::IfLetExprConseqIf{,Let} into AST::IfLetExprConseqElse [[https://github.com/rust-gcc/gccrs/pull/2002][PR2002]]
+- Unify AST::IfExprConseqIf{,Let} into AST::IfExprConseqElse [[https://github.com/rust-gcc/gccrs/pull/2001][PR2001]]
+
+*** Contributors this week
+
+- [[https://github.com/kirmayer-gnu][Sebastian Kirmayer]] (new contributor!)
+- [[https://github.com/P-E-P][Pierre-Emmanuel Patry]]
+- [[https://github.com/powerboat9][Arsen Owens]]
+
+*** Overall Task Status
+
+| Category    | Last Week | This Week | Delta |
+|-------------+-----------+-----------+-------|
+| TODO        |       221 |       220 |    -1 |
+| In Progress |        44 |        44 |     - |
+| Completed   |       623 |       624 |    +1 |
+
+*** Test Cases
+
+The high amount of new testcases is due to the LTO testsuite issue mentioned in previous reports having been fixed by [[https://github.com/tschwinge][Thomas Schwinge]].
+
+| TestCases | Last Week | This Week | Delta |
+|-----------+-----------+-----------+-------|
+| Passing   |   7682    |           |     - |
+| Failed    |   -       |  -        |     - |
+| XFAIL     |   40      |  40       |     - |
+| XPASS     |   -       |  -        |     - |
+
+*** Bugs
+
+| Category    | Last Week | This Week | Delta |
+|-------------+-----------+-----------+-------|
+| TODO        |        70 |        69 |    -1 |
+| In Progress |        19 |        19 |     - |
+| Completed   |       292 |       293 |    +1 |
+
+*** Milestones Progress
+
+We have added more milestones to the project to better compartimentalize the upcoming months of development.
+
+Note that the intrinsics milestone percentage on github is not representative: It shows a 73% completion rate, but does not take into account the tracking issues with dozens of unresolved items.
```

Comment:
I can't see the "73%" completion rate anywhere, this value has surely changed.

Reply:
The value shown on https://github.com/Rust-GCC/gccrs/milestone/9 is 73% completion, but that's not actually the case since it doesn't take into account tracking issues. Unless I'm misunderstanding your comment?